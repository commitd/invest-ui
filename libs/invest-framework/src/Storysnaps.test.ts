import initStoryshots from '@storybook/addon-storyshots'

type GlobalWithAnimationFrame = NodeJS.Global & {
    requestAnimationFrame: (callback: () => {}) => void
}

const polyfilledGlobal = global as GlobalWithAnimationFrame
if (!polyfilledGlobal.requestAnimationFrame) {
    polyfilledGlobal.requestAnimationFrame = (callback: () => {}) => setTimeout(callback, 0)
}

initStoryshots()