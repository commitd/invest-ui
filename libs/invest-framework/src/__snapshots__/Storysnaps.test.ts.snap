// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots FallbackView empty 1`] = `
<div
  className="ui padded four column grid"
/>
`;

exports[`Storyshots FallbackView index 1`] = `
<div
  className="ui padded four column grid"
>
  <div
    className="column"
  >
    <a
      className="ui card"
      href={undefined}
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="search massive icon"
      />
      <div
        className="content"
      >
        <div
          className="header"
        >
          Example
        </div>
        <div
          className="description"
        >
          My example
        </div>
      </div>
    </a>
  </div>
  <div
    className="column"
  >
    <a
      className="ui card"
      href={undefined}
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="search massive icon"
      />
      <div
        className="content"
      >
        <div
          className="header"
        >
          Tester
        </div>
        <div
          className="description"
        >
          Another example
        </div>
      </div>
    </a>
  </div>
</div>
`;

exports[`Storyshots IFrame index 1`] = `
<iframe
  sandbox="allow-scripts allow-forms allow-popups"
  src="http://localhost:6006"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
  title="View"
/>
`;

exports[`Storyshots PluginListSidebar no selection 1`] = `
<div
  className="ui fluid inverted vertical menu"
  style={
    Object {
      "borderRadius": 0,
    }
  }
>
  <a
    className="item"
    onClick={[Function]}
  >
    <h5
      className="ui inverted header"
    >
      <i
        aria-hidden="true"
        className="search icon"
      />
       
      Example
    </h5>
    <small>
      My example
    </small>
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    <h5
      className="ui inverted header"
    >
      <i
        aria-hidden="true"
        className="search icon"
      />
       
      Tester
    </h5>
    <small>
      Another example
    </small>
  </a>
</div>
`;

exports[`Storyshots PluginListSidebar with selected 1`] = `
<div
  className="ui fluid inverted vertical menu"
  style={
    Object {
      "borderRadius": 0,
    }
  }
>
  <a
    className="active item"
    onClick={[Function]}
  >
    <h5
      className="ui inverted header"
    >
      <i
        aria-hidden="true"
        className="search icon"
      />
       
      Example
    </h5>
    <small>
      My example
    </small>
  </a>
  <a
    className="item"
    onClick={[Function]}
  >
    <h5
      className="ui inverted header"
    >
      <i
        aria-hidden="true"
        className="search icon"
      />
       
      Tester
    </h5>
    <small>
      Another example
    </small>
  </a>
</div>
`;

exports[`Storyshots PluginView with selected 1`] = `
<iframe
  sandbox="allow-scripts allow-forms allow-popups"
  src="http://localhost:6006/"
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
  title="View"
/>
`;

exports[`Storyshots PluginViewManager empty 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots PluginViewManager no selection 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <p>
    has a fallback
  </p>
</div>
`;

exports[`Storyshots PluginViewManager no selection, no fallback 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;

exports[`Storyshots PluginViewManager with selected 1`] = `
<div
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <div
    style={
      Object {
        "display": "block",
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <iframe
      sandbox="allow-scripts allow-forms allow-popups"
      src="http://localhost:6006/"
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
      title="View"
    />
  </div>
</div>
`;

exports[`Storyshots Welcome Invest Framework 1`] = `
<div>
  <p>
    Invest Framework holds components which are useful for building the the outer frame application rather than than for plugins.
  </p>
</div>
`;
